.github/workflows/broken-link-check-schedule.yml

```
@@ -0,0 +1,37 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+# Based on hashicorp/consul/.github/workflows/broken-link-check.yml
+# Run lychee link check every Sat at 2am
+# Create GH issue from lychee output; assign issue to Aimee Ukasick
+
+name: Broken Link Check Scheduled
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 2 * * SAT" # every Sat at 2 am
+
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4.1.4
+
+      - name: Run lychee link checker
+        id: lychee
+        uses: lycheeverse/lychee-action@2b973e86fc7b1f6b36a93795fe2c9c6ae1118621 # v1.10.0
+        with:
+          args: ./website/content/docs/ --base https://developer.hashicorp.com/ --exclude-all-private --exclude '\.(svg|gif|jpg|png)' --exclude 'manage\.auth0\.com' --accept 403 --max-concurrency=24 --no-progress --verbose
+          # Fail GitHub action when broken links are found?
+          fail: false
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Seeing as how we want this workflow to _write_ to the repo (to create an issue), tokens can get a little complicated because of the CE->ENT merge. Given that we're only running this once a week, we may want to consider landing this PR only in the ENT repo so that we only have to make it work in one place.

```suggestion
          GITHUB_TOKEN: ${{ secrets.ELEVATED_GITHUB_TOKEN }}
```